# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/socket/nsSOCKSIOLayer.cpp
# Commit: 48eaa84c206d
# Full Hash: 48eaa84c206dbdda637c57b55ddbd07e975ba103
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1637500
# File Overlap Count: 2
# Description:
#   Bug 1637500 - P2. Rename methods as they are not always dealing with "threads". r=froydnj
#   
#   Before P1, GetCurrentThreadSerialEventTarget would have always returned the same data as NS_GetCurrentThread, making the comment incorrect Now it will properly return the running TaskQueue if any.
#   
#   This change of name more clearly exposes what they are doing, as we aren't always dealing with threads directly; but a nsISerialEventTarget
# ==============================================================================

diff -r 00687d06a8a6 -r 48eaa84c206d netwerk/socket/nsSOCKSIOLayer.cpp
--- a/netwerk/socket/nsSOCKSIOLayer.cpp	Tue Jun 23 05:02:39 2020 +0000
+++ b/netwerk/socket/nsSOCKSIOLayer.cpp	Tue Jun 23 05:05:36 2020 +0000
@@ -469,8 +469,8 @@
 
   mFD = fd;
   nsresult rv = dns->AsyncResolveNative(proxyHost, 0, this,
-                                        mozilla::GetCurrentThreadEventTarget(),
-                                        attrs, getter_AddRefs(mLookup));
+                                        mozilla::GetCurrentEventTarget(), attrs,
+                                        getter_AddRefs(mLookup));
 
   if (NS_FAILED(rv)) {
     LOGERROR(("socks: DNS lookup for SOCKS proxy %s failed", proxyHost.get()));